### DISCUSSION TITLE
3020 sff using igpu no display after reconnecting monitor

### DISCUSSION THREAD

**Discussion by eegah** [varszegimarcell/Optiplex-3020-Hackintosh-OpenCore#60] [Q&A]
I have an issue with display.  if i turn off the monitor or disconnect the cable, upon reconnecting there is no display.  I have to do a manual reboot.  any ideas? im using everything default from this efi besides of course my machine specific info added to the config.

**Comment by Xtendera** [comment]
What is your CPU model?

**Reply by eegah** [reply]
i5 4590

**Comment by Xtendera** [comment]
Meaning that it isn't a Graphics card issue. 

Hmmm...

**Reply by eegah** [reply]
sorry for the delay, yeah its not a card issue.  i have 2 of these, one with a gt 740 and there is no issue with reconnecting via the card on that one.  just the i5 sans card.

**Comment by Scott-GT** [comment]
I was having a similar issue but after updating the OS to 12.6, OC0.8.5 and the latest kexts as of this week...the issue has gone. I'm not sure which of these has resolved it but I would have thought the latest kexts may have helped.
